.github/workflows/post-merge-actions-build.yml

```diff
@@ -0,0 +1,84 @@
+name: Build Node server build and deploy
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./
+
+# To be used when the container signer stack is setup through the Terraform code in di-ipv-cri-pipeline-deployment
+# jobs:
+#   deploy:
+#     runs-on: ubuntu-latest
+#     timeout-minutes: 60
+#     permissions:
+#       id-token: write
+#       contents: read
+#     steps:
+#       - name: Upload to ECR and tag
+#         uses: alphagov/di-devplatform-upload-action-ecr@1.0.1
+#         with:
+#           role-to-assume-arn: ${{ secrets.BUILD_ROLE_TO_ASSUME }}
+#           container-sign-kms-key-arn: ${{ secrets.BUILD_CONTAINER_SIGN_KMS_KEY }}
+#           ecr-repo-name: ${{ secrets.BUILD_ECR_REPOSITORY }}
+#           artifact-bucket-name: ${{ secrets.BUILD_ARTIFACT_BUCKET }}
+
+jobs:
+  dockerBuildAndPush:
+    name: Docker build and push
+    runs-on: ubuntu-latest
+    env:
+      AWS_REGION: eu-west-2
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: '0'
+
+      - name: Set up AWS creds
+        uses: aws-actions/configure-aws-credentials@v1-node16
```

Comment:
Are we able to use `aws-actions/configure-aws-credentials@v2` ?

Reply:
I don't see why not - out of curiosity what would the benefit be? And why not a higher version?

Reply:
Done